default/stringtables/en.txt

```diff
@@ -15434,6 +15434,9 @@ Protection Focus
 GASEOUS_LABEL
 Gas Giant Environment
 
+MINDLESS_LABEL
+Mindless Species
```

Comment:
If there is some fluff for the effect (why is mindless species decreasing population), this entry should be changed accordingly.
Bad suggestion: Mindless Wandering Off

Reply:
I'm thinking of making them bad population and very bad (50%) at industry and research, instead of this "mindless" thingy I came up with:
- Easier fluff.
- Slightly better balance (currently they are still better than many other species in poor planets until mid-late game).
- Makes them compatible with the future intended change of making good and adequate environments more valuable.